reference/docs-conceptual/community/contributing/quality-improvements.md

```diff
@@ -0,0 +1,239 @@
+to improve. Once you've decided what articles and content areas you want to work on, follow these
+steps:
+
+<!-- markdownlint-disable MD044 -->
+
+1. Check the [project][23] for issues filed for this effort to avoid duplicating efforts.
+1. Open a new issue in the appropriate repository:
+   - Open an issue in [MicrosoftDocs/PowerShell-Docs][20] for PowerShell reference and conceptual
+     content.
+   - Open an issue in [MicrosoftDocs/PowerShell-Docs-Dsc][16] for DSC content
+   - Open an issue in [MicrosoftDocs/PowerShell-Docs-Modules][18] for Crescendo, PlatyPS,
+     PSScriptAnalyzer, SecretManagement, and SecretStore content.
+   - Open an issue in [MicrosoftDocs/azure-docs-powershell][13] for Azure PowerShell content.
+   - Open an issue in [MicrosoftDocs/windows-powershell-docs][22] for Windows PowerShell module
+     content.
+1. Follow our [contributor's guide][01] to get setup for making your changes.
+1. Submit your pull request. Ensure:
+
+   1. Your PR title has the `Quality:` prefix.
+   1. Your PR body references the issue it resolves as an unordered list item and uses one of the
+      [linking keywords][10].
+
+      For example, if you're working on issue `123`, the body of your PR should include the
+      following Markdown:
+
+      ```md
+      - resolves #123
+      ```
+
+   The content developers will review your work as soon as they can to help you get it merged.
+
+<!-- markdownlint-enable MD044 -->
+
+<!-- Link References -->
+[01]: /powershell/scripting/community/contributing/overview#prepare-to-make-a-contribution
+[02]: /powershell/scripting/community/contributing/powershell-style-guide#formatting-command-syntax-elements
+[03]: /powershell/scripting/community/contributing/powershell-style-guide#markdown-for-code-samples
+[04]: #aliases
+[05]: #formatting-code-samples
+[06]: #formatting-command-syntax
+[07]: #link-references
+[08]: #markdown-linting
+[09]: #spelling
+[10]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+[11]: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022
+[12]: https://github.com/MicrosoftDocs/azure-docs-powershell
+[13]: https://github.com/MicrosoftDocs/azure-docs-powershell/issues/new?template=00-hacktoberfest.yml&title=Quality%3A+
+[14]: https://github.com/MicrosoftDocs/PowerShell-Docs
+[15]: https://github.com/MicrosoftDocs/PowerShell-Docs-Dsc
+[16]: https://github.com/MicrosoftDocs/PowerShell-Docs-Dsc/issues/new?template=00-hacktoberfest.yml&title=Quality%3A+
+[17]: https://github.com/MicrosoftDocs/PowerShell-Docs-Modules
+[18]: https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/issues/new?template=00-hacktoberfest.yml&title=Quality%3A+
+[19]: https://github.com/MicrosoftDocs/PowerShell-Docs/issues/9257
+[20]: https://github.com/MicrosoftDocs/PowerShell-Docs/issues/new?template=00-hacktoberfest.yml&title=Quality%3A+
```

Comment:
Shouldn't we point to the new template?

May want to change this for all repos now even though you haven't put the new template in place. Saves doing iterative updates each time to update another repo.

Reply:
good catch, will update!